src/components/OSCALLoaderForm.js

```diff
@@ -63,7 +61,7 @@ export default function OSCALLoaderForm(props) {
       }}
     >
       <Grid container spacing={3}>
-        {process.env.REACT_APP_REST_BASE_URL && (
+        {!!props.backendUrl && (
```

Comment:
Any reason we're using `!!`?

Reply:
Good question! Mostly I'm scared of short-circuiting and non-explicit truthiness checks so I wanted to coerce it to a boolean. I suppose that without `!!` the condition will always be the same result? If you say it will be then I'll go ahead and remove the `!!`. I had a concern that there might be a scenario where we actually displayed `""` if `REACT_APP_REST_BASE_URL` was set but it was set to the empty string.

Reply:
I'm not concerned about empty strings, as removing the `!!` will still lead to the same result. I'm more concerned if someone provides a string containing only whitespace, which the `!!` handles incorrectly anyways.

Reply:
True; though we never really validate the REST API URL anyway. So `" "` isn't any different than `"https://google.com/"`. I'll tinker with it and see if I can find a better solution here.